chirp/drivers/retevis_ha1g.py

```diff
@@ -1265,6 +1340,8 @@ class HA1G(chirp_common.CloneModeRadio):
     write_page_len = 1024
     current_model = "HA1G"
     _ch_cache = None
+    _dtmf_list = None
+    _alarm_list = None
```

Comment:
These should probably default to empty lists since you treat them as such elsewhere. Don't assume you'll always have run `process_mmap()` before they are used.